### DISCUSSION TITLE
How do you setup an OpenAI model from Azure AI?

### DISCUSSION THREAD

**Discussion by j-taptok** [open-webui/open-webui#1114] [Q&A] [answered]
Changelog shows that you can integrate with OpenAI API and compatible services, Azure AI I believe is compatible. Which endpoint I need to setup? The direct model conversations endpoint or the AI project endpoint?

Tried both but nothing happened.

<img width="966" alt="image" src="https://github.com/open-webui/open-webui/assets/98992496/108243e5-e895-46a3-8bfd-f06b6e842637">

**Comment by justinh-rahb** [comment, marked as answer]
Have a look here, we've added new features and abilities since that post you reference.

- #1038

**Reply by j-taptok** [reply]
you are awesome! thanks 
<img width="885" alt="image" src="https://github.com/open-webui/open-webui/assets/98992496/f611933f-d8fd-4e4b-be90-527f38ec3cfd">
